flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/dataset/DataSetWindowAggregateITCase.scala

```diff
@@ -354,4 +354,94 @@ class DataSetWindowAggregateITCase(
     val results = windowedTable.toDataSet[Row].collect()
     TestBaseUtils.compareResultAsText(results.asJava, expected)
   }
+
+  @Test
+  def testEventTimeSlidingGroupWindowOverCountOverlappingFullPane(): Unit = {
+    // please keep this test in sync with the DataStream processing-time variant
+    val env = ExecutionEnvironment.getExecutionEnvironment
+    val tEnv = TableEnvironment.getTableEnvironment(env)
+
+    val table = env
+      .fromCollection(data)
+      .toTable(tEnv, 'long, 'int, 'double, 'float, 'bigdec, 'string)
+
+    val windowedTable = table
+      .window(Slide over 4.rows every 2.rows on 'long as 'w)
+      .groupBy('string, 'w)
+      .select('string, 'int.count)
+
+    val expected =
+      "Hello world,2\n" +
+      "Hello,2"
```

Comment:
shouldn't this be "Hello,4"? There are four rows with `'string = "Hello"` in the data set. Shouldn't those be aggregated in a sliding count window of size 4?